components/authz-service/server/v2/policy.go

```diff
@@ -154,18 +154,24 @@ func (s *policyServer) CreatePolicy(
 			"error parsing policy %q: %s", req.Id, err.Error())
 	}
 
-	returnPol, err := s.store.CreatePolicy(ctx, &pol)
+	returnPol, err := s.store.CreatePolicy(ctx, &pol, s.isBeta2p1())
 	switch err {
 	case nil: // continue
 	case storage_errors.ErrConflict:
 		return nil, status.Errorf(codes.AlreadyExists,
 			"policy with id %q already exists", req.Id)
 	default:
-		if err, ok := err.(*storage_errors.ForeignKeyError); ok {
+		switch err.(type) {
+		case *projectassignment.ProjectsMissingErr:
```

Comment:
Let's follow the [error naming convention](https://github.com/golang/go/wiki/Errors#naming), please:

> Error types end in "Error" and error variables start with "Err" or "err"

So, 
```suggestion
		case *projectassignment.ProjectsMissingError:
```
etc

Reply:
TIL